### ISSUE TITLE
Listening to?

### ISSUE THREAD

**Issue by Dolinmyster** [ToppleKek/discord-rhythmbox-plugin#16]
Hello,

I wanted to say that I am really enjoying this plug in so far. I wanted to see if it would be possible to change it from "Playing a Game" to "Listening To" on the Discord side. It appears that it is possible from the documentation I have looked at with Discord, but sadly my programming knowledge is limited to outdated HTML 4 and SQL Query. Thank you in advance! And I hope this is an easy change to make.

**Comment by byemc** [comment]
"Listening to" only works with Spotify with their special integration with Discord. iirc RPC can't take advantage of this

**Comment by byemc** [comment]
You might be looking at the documentation for bots though, because you *can* do this with bots. But as far as I know, rich presence can only be **Playing**.

**Comment by Dolinmyster** [comment]
I have seen it done with Bots. I wasn't sure if that was something that can be done through utilities like this. It seemed like the Activity Enum would be able to do it. If this is bot only stuff, as I know my instance of Red can do it, then that is fine. Just wanted to see if it was possible. Thanks again! I do love the plugin. It has been awesome. 

https://discord.com/developers/docs/game-sdk/activities#data-models-activitytype-enum

**Comment by byemc** [comment]
> I have seen it done with Bots. I wasn't sure if that was something that can be done through utilities like this. It seemed like the Activity Enum would be able to do it. If this is bot only stuff, as I know my instance of Red can do it, then that is fine. Just wanted to see if it was possible. Thanks again! I do love the plugin. It has been awesome.
> 
> https://discord.com/developers/docs/game-sdk/activities#data-models-activitytype-enum

Yeah, it's a great plug in. It is bot only, otherwise im sure the **Listening to** thing would've been implemented already.

**Comment by ToppleKek** [comment]
Yes, this can be done with bots and even games. As your link suggests, this feature is part of the Discord game SDK. This is a SDK targeted at game developers so that they can better implement things like rich presence, game invites, etc. So technically it is possible for us to implement this by creating a "game" and having the plugin communicate with it, but honestly it seems like a lot of work just to have it say "Listening to" on your profile :)
